sdk/appcenter-crashes/src/test/java/com/microsoft/appcenter/crashes/CrashesTest.java

```diff
@@ -1120,19 +1129,23 @@ public ManagedErrorLog answer(InvocationOnMock invocation) {
         verify(mockChannel, never()).enqueue(any(ManagedErrorLog.class), eq(crashes.getGroupName()), anyInt());
     }
 
+    @SuppressWarnings("ResultOfMethodCallIgnored")
```

Comment:
Can assert true in tests instead of suppressing.

Reply:
Thank you, fixed.